### PR TITLE
Properly transform youtu.be links to be /watch routes in comments and descriptions

### PR DESCRIPTION
**Pull request by syeopite**
Follow up to https://github.com/iv-org/invidious/issues/2347#issuecomment-908884819

### CODE REVIEW

File: src/invidious/comments.cr [hunk 1]
```diff
@@ -552,7 +552,10 @@ def content_to_comment_html(content)
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
         url = URI.parse(url)
 
-        if !url.host || {"m.youtube.com", "www.youtube.com", "youtu.be"}.includes? url.host
+        if !url.host
+        elsif url.host == "youtu.be"
+          url = "/watch?v=#{url.request_target.lstrip('/')}"
+        elsif {"m.youtube.com", "www.youtube.com"}.includes? url
           if url.path == "/redirect"
```
Comment (syeopite) [inline src/invidious/comments.cr:559]: Afaik `youtu.be` doesn't actually have a `/redirect` path. It should be fine to remove handling for `youtu.be` here.

File: src/invidious/comments.cr [hunk 2]
```diff
@@ -552,7 +552,10 @@ def content_to_comment_html(content)
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
         url = URI.parse(url)
 
-        if !url.host || {"m.youtube.com", "www.youtube.com", "youtu.be"}.includes? url.host
+        if !url.host
+        elsif url.host == "youtu.be"
+          url = "/watch?v=#{url.request_target.lstrip('/')}"
+        elsif {"m.youtube.com", "www.youtube.com"}.includes? url
```
Comment (SamantazFox) [inline src/invidious/comments.cr:558]:
```suggestion
        elsif {"m.youtube.com", "www.youtube.com"}.includes? url.host
```

Comment (syeopite) [reply src/invidious/comments.cr:558]: Ah thanks! Didn't realize I removed it.

File: src/invidious/comments.cr [hunk 3]
```diff
@@ -552,7 +552,10 @@ def content_to_comment_html(content)
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
         url = URI.parse(url)
 
-        if !url.host || {"m.youtube.com", "www.youtube.com", "youtu.be"}.includes? url.host
+        if !url.host
```
Comment (SamantazFox) [inline src/invidious/comments.cr:555]: If I'm not mistaken, `if !url.host` should be handled the same way as if the domain is `*.youtube.com`, in the case where google uses relative links

Comment (syeopite) [reply src/invidious/comments.cr]: Oops, thanks!

**Review by SamantazFox — APPROVED** [review]

**Pull request merged**